sdk/core/core-rest-pipeline/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@azure/core-rest-pipeline",
-  "version": "1.13.1",
+  "version": "1.13.0",
```

Comment:
Is this a mistake?

Reply:
Fix. The change was on top of an old commit.